spec/cloud/google_service_spec.rb

```diff
@@ -603,6 +605,18 @@
     end
   end
 
+  describe '#service_account_scope_url' do
+    it "returns the passed-in scope if it already looks like a scope URL" do
+      scope = "https://www.googleapis.com/auth/fake_scope"
+      expect(service.service_account_scope_url(scope)).to eq(scope)
+    end
+
+    it "returns a properly-formatted scope URL if a short-name or alias is provided" do
+      expect(service).to receive(:translate_scope_alias).with("scope_alias").and_return("real_scope")
```

Comment:
It's not in the map. That's what this mock does... it's not testing the map, it's testing that `#service_account_scope_url` calls `#translate_scope_alias` and gets a response, and that response is then used in the return to the caller.

Remember unit testing is testing the messages sent and received. I'm not actually testing the data in the alias map - I'm testing that the methods call the right methods and use the responses appropriately.